src/components/task-plan/homework/exercise-controls.cjsx

```diff
@@ -68,24 +68,28 @@ ExerciseControls = React.createClass
     else if @props.canAdd
       <BS.Button bsStyle="default"
         className="-add-exercises"
-        onClick={@props.addClicked}>Add More...</BS.Button>
+        onClick={@props.addClicked}>+ Add More Sections</BS.Button>
     else
       null
 
   canChangeTutorQty: ->
     @props.canEdit or @props.canAdd
 
   renderIncreaseButton: ->
-    return null unless @canChangeTutorQty() and TaskPlanStore.canIncreaseTutorExercises(@props.planId)
-    <BS.Button onClick={@addTutorSelection} className="btn-xs circle">
-      <Icon type='arrow-up' />
-    </BS.Button>
+    if @canChangeTutorQty() and TaskPlanStore.canIncreaseTutorExercises(@props.planId)
+      <BS.Button onClick={@addTutorSelection} className="btn-xs hover-circle">
+        <Icon type='chevron-up' />
+      </BS.Button>
+    else
+      <span className="circle-btn-placeholder"></span>
 
   renderDecreaseButton: ->
-    return null unless @canChangeTutorQty() and TaskPlanStore.canDecreaseTutorExercises(@props.planId)
-    <BS.Button onClick={@removeTutorSelection} className="btn-xs circle">
-      <Icon type='arrow-down' />
-    </BS.Button>
+    if @canChangeTutorQty() and TaskPlanStore.canDecreaseTutorExercises(@props.planId)
+      <BS.Button onClick={@removeTutorSelection} className="btn-xs hover-circle">
+        <Icon type='chevron-down' />
+      </BS.Button>
+    else
+      <span className="circle-btn-placeholder"></span>
 
```

Comment:
A nitpick, but React allows you to just do  XML style empty tags `<span className="circle-ban-placeholder"/>`